app/src/utils/formatDate/formatDate.ts

```diff
@@ -0,0 +1,12 @@
+function formatDate(date: string) {
+  if (!date) return '';
+
+  const dateToFormat = new Date(date);
+
+  return new Intl.DateTimeFormat('en-GB', {
```

Comment:
We could create a reusable formatDate for language localization in this case, our user is probably not from England, so we could pass the language code as second parameter in order to adapt each case